File: nodecore-spv/src/main/java/veriblock/model/DownloadStatus.java
package veriblock.model;

public enum DownloadStatus {
    DISCOVERING,
    DOWNLOADING,
    READY;

    public boolean isDiscovering(){
        return DownloadStatus.DISCOVERING == this;
    }
    public boolean isDownloading(){
        return DownloadStatus.DOWNLOADING == this;
    }
}

File: nodecore-spv/src/main/java/org/veriblock/spv/model/DownloadStatus.kt
package org.veriblock.spv.model

enum class DownloadStatus {
    DISCOVERING,
    DOWNLOADING,
    READY;

    fun isDiscovering(): Boolean =
        DISCOVERING == this

    fun isDownloading(): Boolean =
        DOWNLOADING == this

    fun isReady(): Boolean =
        READY == this
}
